paddle/fluid/operators/multinomial_op.cc

```diff
@@ -53,12 +53,25 @@ class MultinomialOp : public framework::OperatorWithKernel {
 
     auto x_dim = ctx->GetInputDim("X");
     int64_t x_rank = x_dim.size();
+    PADDLE_ENFORCE_GT(x_rank, 0, platform::errors::PreconditionNotMet(
+                                     "Input probability distribution should be "
+                                     "1 or 2 dimension, but got %d",
+                                     x_rank));
+    PADDLE_ENFORCE_LE(x_rank, 2, platform::errors::PreconditionNotMet(
+                                     "Input probability distribution should be "
```

Comment:
```suggestion
                                     "The number of  dimensions of  the input probability distribution should be <= 2, but got %d."
```
Similar for the others.

Reply:
done